mllib/src/test/scala/org/apache/spark/ml/evaluation/RegressionEvaluatorSuite.scala

```diff
@@ -65,15 +65,15 @@ class RegressionEvaluatorSuite
 
     // default = rmse
     val evaluator = new RegressionEvaluator()
-    assert(evaluator.evaluate(predictions) ~== 0.1019382 absTol 0.001)
+    assert(evaluator.evaluate(predictions) ~== 0.1013829 absTol 0.001)
```

Comment:
@srowen I think the difference is more - the original tolerance were based on the idea that this should match the R implementation values closely, but since we've changed the data (still has the same global distribution) the exact value is a little different - so its a question of if we want the tolerance to be based on R predicting on the same data set or the tolerance to be based on what it should look like for a model trained on data with this distribution. Does that sound correct or am I out in left field?